lib/builtin.pm

```diff
@@ -371,6 +372,21 @@ This must be called at compile time; which typically means during a C<BEGIN>
 block. Usually this would be used as part of an C<import> method of a module,
 when invoked as part of a C<use ...> statement.
 
+=head2 load_module
+
+    load_module($module_name);
```

Comment:
As it's new (and therefore experimental) don't forget to copy the "This function is experimental" text from one of the other function docs.

Reply:
Done that in latest commit.